rcdk/R/rcdk.R

```diff
@@ -80,18 +80,31 @@ get.total.hydrogen.count <- function(mol) {
 get.exact.mass <- function(mol) {
   if (!.check.class(mol, "org/openscience/cdk/interfaces/IAtomContainer"))
     stop("molecule must be of class IAtomContainer")
-  ret <- .jcall('org/openscience/cdk/tools/manipulator/AtomContainerManipulator',
+  
+  
+  formulaJ <- .jcall('org/openscience/cdk/tools/manipulator/MolecularFormulaManipulator',
+                     "Lorg/openscience/cdk/interfaces/IMolecularFormula;",
+                     "getMolecularFormula",
+                     molecule,
```

Comment:
Hi, this can't work, the function argument is 'mol'. Can be fixed with:
```suggestion
                     mol,
```